app/js/actions/fetchOrders.js

```diff
@@ -9,7 +9,7 @@ const fetchOrders = (
 ) => ({
   type: FETCH_ORDERS,
   payload: axiosInstance
-    .get(uri || `/order?totalCount=true&sort=desc&status=active&patient=${patient}&limit=${limit}&v=full`),
+    .get(uri ? uri.substr(uri.indexOf('v1') + 2) : `/order?totalCount=true&sort=desc&status=active&patient=${patient}&limit=${limit}&v=full`),
```

Comment:
Why index of v1?

Reply:
this gets the index of v1, then we add 2. In other to remove `http://localhost:8081/openmrs/ws/rest/v1`  or  `https://demo.openmrs.org/openmrs/ws/rest/v1`
There is only one v1 in the string. This helps us remove unwanted string effectively.